tests/v2/validation/configmaps/generated_fields_test.go

```diff
@@ -0,0 +1,174 @@
+package configmaps
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	steveV1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeapi/configmaps"
+	"github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+	v1 "k8s.io/api/core/v1"
+	"net/http"
+	"strings"
+	"testing"
+)
+
+const (
+	APIVersion = "v1"
+	Kind       = "ConfigMap"
+)
+
+type ConfigMapTestSuite struct {
+	suite.Suite
+	client           *rancher.Client
+	steveClient      *steveV1.Client
+	session          *session.Session
+	cluster          *management.Cluster
+	httpClient       *http.Client
+	configMapPayload *configMapPayload
+}
+
+type configMapPayload struct {
```

Comment:
I do not have a strong opinion but do you think we can add this as a template like [here](https://github.com/rancher/rancher/blob/release/v2.7/tests/framework/extensions/workloads/template.go ) in configMaps extension [here](https://github.com/rancher/rancher/tree/release/v2.7/tests/framework/extensions/configmaps) and use that extension?

Reply:
Sounds good! Moved.